pom.xml

```diff
@@ -1475,6 +1477,12 @@
                 <artifactId>threeten-extra</artifactId>
                 <version>${threeten.version}</version>
             </dependency>
+
+            <dependency>
+                <groupId>org.apache.hudi</groupId>
+                <artifactId>hudi-spark${hudi.spark.binary.version}-bundle_2.12</artifactId>
```

Comment:
@AngersZhuuuu since we have profile `hudi` to control the importing of hudi bundle jar, is it good to use `scala_binary_version` here now?

Reply:
> @AngersZhuuuu since we have profile `hudi` to control the importing of hudi bundle jar, is it good to use `scala_binary_version` here now?

Yea. Can use `scala_binary_version ` now

Reply:
Done